### DISCUSSION TITLE
Run old versions of Maputnik

### DISCUSSION THREAD

**Discussion by zstadler** [maputnik/editor#887] [Ideas]
**Maputnik version**: main
**Browser**: All
**OS**: All

**Description of the bug**:
The [Troubleshooting](../wiki/Troubleshooting) wiki page refers to the ability to run previous versions of Maputnik hosted at `https://maputnik.github.io/releases/[VERSION]`, such as https://maputnik.github.io/releases/v1.7.0

The previous releases exist at https://github.com/maplibre/maputnik/releases, it is not clear what is the MapLibre equivalent of `https://maputnik.github.io/releases/[VERSION]`.

**Comment by HarelM** [comment]
It's a good question.
My initial idea was that this is a CI/CD SaaS type of service where every commit to main is published automatically, meaning there's no real "versions".
I'm guessing that if someone would like to build an "old" version it can be done by taking the repo in the relevant commit and building it locally.
The question that comes to my mind is what the problem these version are trying to solve? Why do "versions"?

**Comment by zstadler** [comment]
I assume it's coming from a humble view that the latest commit can introduce bugs or incompatibilities that did not exist before, and that even "official" releases may have an occasional glitch.

I also believe that Maputnik users should not be expected to build it.  Such a mechanism of making the previous versions available is a simple low cost alternative. 

Perhaps @orangemug can shed some light on this.

**Comment by HarelM** [comment]
In a SaaS type of offering the direction is forward, if someone would like to add support for versioning and all that, feel free to do so, maintenance of this is already stretched.
